invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/Identifiers/PIDField/components/UnmanagedIdentifierCmp.js

```diff
@@ -0,0 +1,80 @@
+// This file is part of Invenio-RDM-Records
+// Copyright (C) 2020-2023 CERN.
+// Copyright (C) 2020-2022 Northwestern University.
+//
+// Invenio-RDM-Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { Form } from "semantic-ui-react";
+import { getFieldErrors } from "./helpers";
+
+/**
+ * Render identifier field to allow user to input
+ * the unmanaged PID.
+ */
+export class UnmanagedIdentifierCmp extends Component {
+  constructor(props) {
+    super(props);
+
+    const { identifier } = props;
+
+    this.state = {
+      localIdentifier: identifier,
```

Comment:
Question: I might be missing something, but I don't see added value on adding the usage of `localIdentifier`, couldn't we rely on the prop `identifier` alone?

Reply:
I don't think the component would be updated i.e. there will not be a state update to force a rerender if we rely only on the `props.identifier`.